packages/contracts/test/plugins/governance/multisig/multisig.ts

```diff
@@ -258,6 +260,36 @@ describe('Multisig', function () {
       );
       expect(toProtocolVersion).to.not.deep.equal(CURRENT_PROTOCOL_VERSION);
     });
+
+    it('from v1.3.0', async () => {
+      legacyContractFactory = new Multisig_V1_3_0__factory(signers[0]);
+
+      const {fromImplementation, toImplementation} =
+        await deployAndUpgradeFromToCheck(
+          signers[0],
+          signers[1],
+          initArgs,
+          'initialize',
+          legacyContractFactory,
+          currentContractFactory,
+          UPGRADE_PERMISSIONS.UPGRADE_PLUGIN_PERMISSION_ID,
+          dao
+        );
+      expect(toImplementation).to.equal(fromImplementation);
+
+      const fromProtocolVersion = await getProtocolVersion(
+        legacyContractFactory.attach(fromImplementation)
+      );
+      const toProtocolVersion = await getProtocolVersion(
+        currentContractFactory.attach(toImplementation)
+      );
+
+      expect(fromProtocolVersion).to.deep.equal(toProtocolVersion); // The contracts inherited from OSx did not change from 1.3.0 to the current version
+      expect(fromProtocolVersion).to.deep.equal(
+        IMPLICIT_INITIAL_PROTOCOL_VERSION
+      );
+      expect(toProtocolVersion).to.not.deep.equal(CURRENT_PROTOCOL_VERSION);
+    });
```

Comment:
I am a bit confused here.
since there is no new build for the multisig, why we have this test?
what actually r we testing here?

Reply:
We have this test because the OZ dependencies have changed.
This test that the upgrade from 1.3.0 to 1.4.0 (where we use a newer OZ) still works (and that we don't accidentally break storage). This is unexpected. Still our practice is testing each upgrade we do (see our [UPDATE_CHECKLIST.md](https://github.com/aragon/osx/blob/b27c084d83427bbbcb4cba81080356ae3c8aedca/UPDATE_CHECKLIST.md?plain=1#L9)).